spec/mailers/user_mailer_spec.rb

```diff
@@ -19,8 +19,8 @@
     let(:mail) { UserMailer.admin_new_service_provider(app) }
 
     it 'sends to ADMIN_EMAIL' do
-      ClimateControl.modify ADMIN_EMAIL: 'identity-admin@example.com' do
-        expect(mail.to).to eq ['identity-admin@example.com']
+      ClimateControl.modify ADMIN_EMAIL: 'partners@login.gov' do
```

Comment:
do we have to change this to a potentially real email? I think we were trying to be compliant with https://tools.ietf.org/html/rfc2606 here